bin/install-test-env.sh

```
@@ -116,9 +116,9 @@ if [ "$WP_DEBUG" != $WP_DEBUG_CURRENT ]; then
 	WP_DEBUG_RESULT=$(wp config get --type=constant --format=json WP_DEBUG  --allow-root | tr -d '\r')
 	echo -e "$(status_message "WP_DEBUG: $WP_DEBUG_RESULT...")"
 
-	# @todo Unmute WP_DEBUG_DISPLAY
-	# @see https://github.com/wp-graphql/wp-graphql/issues/3239
-	wp config set WP_DEBUG_DISPLAY false --raw --type=constant --quiet --allow-root
+	wp config set WP_DEBUG_DISPLAY true --raw --type=constant --quiet --allow-root
```

We don't need to _change_ these lines, we want to _remove_ them aka how it was before we had to temporarily suppress the errors: https://github.com/rtCamp/snapwp-helper/commit/8a5932481a8976f137f693f633feebf00fb896ea